formats/calendar/SRF_Calendar.php

```diff
@@ -357,7 +367,12 @@ function displayCalendar( $events ) {
 		}
 
 		$cur_month = self::intToMonth( $cur_month_num );
-		$cur_year = date( 'Y', time() );
+
+		if ( is_numeric( $this->mStartYear ) && ( intval( $this->mStartYear ) == $this->mStartYear ) ) {
```

Comment:
Hi mwjames, thanks for having a look into this. I did a test with mStartYear = -2000 and it works fine (not crashing the calendar). I assume that this is interpreted as 2000 years before the Nativity. If no year is specified it still sets $cur_year = date( 'Y', time() );